packages/core/src/layout/InfoCard/InfoCard.tsx

```diff
@@ -142,6 +150,7 @@ const InfoCard: FC<Props> = ({
   actions,
   cardClassName,
   actionsTopRight,
+  noPadding = true,
```

Comment:
I don't think we want this to be true by default. Better leave this as false and then use it in the Status component story.

Reply:
Alright!
I just assumed that this is the default design based on https://github.com/spotify/backstage/issues/830